### PR TITLE
Fix transaction history scroll and add loading

### PR DESCRIPTION
**Pull request by jopedroliveira**
Fix transaction vertical scroll
Feat loading animation while waiting for backend

### CODE REVIEW

File: main/station-config.js [hunk 1]
```diff
@@ -124,11 +125,67 @@ function getStationWalletBalance () {
   return 0 // todo - backend logic
 }
 
+function firstFunction () {
```
Comment (juliangruber) [inline main/station-config.js:128]: what does `firstFunction` mean?

File: main/station-config.js [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 const Store = require('electron-store')
 const { randomUUID } = require('crypto')
+const { resolve } = require('path')
```
Comment (juliangruber) [inline main/station-config.js:5]: what is this for?

File: renderer/src/components/WalletTransactionsHistory.tsx [hunk 3]
```diff
@@ -4,23 +4,29 @@ import dayjs from 'dayjs'
...
 import { ReactComponent as OutcomeIcon } from '../assets/img/icons/outcome.svg'
 import { ReactComponent as ExternalLinkIcon } from '../assets/img/icons/external.svg'
+import { ReactComponent as LoadingIcon } from '../assets/img/icons/loading.svg'
 import WalletTransactoinStatusWidget from './WalletTransactionStatusWidget'
 import { brownseTransactionTracker } from '../lib/station-config'
 import WalletOnboarding from './WalletOnboarding'
 
 interface WalletTransactionsHistoryProps {
-  allTransactions: FILTransaction[] | [],
+  allTransactions: FILTransaction[] | [] | null,
```
Comment (juliangruber) [inline renderer/src/components/WalletTransactionsHistory.tsx:13]:
```suggestion
  allTransactions: FILTransaction[] | null,
```

Looks like the empty array doesn't provide any value as a type here

**Comment by juliangruber** [comment]
Screenshot 2023-01-17 at 10 44 54

Is it intentional to display the loading animation and onboarding at the same time?

**Review by juliangruber — CHANGES_REQUESTED** [review]
Also this PR is still a draft, shall we mark it ready for review?

File: main/station-config.js (hunk 2 above)
Comment (jopedroliveira) [reply main/station-config.js:5]: It's a draft. I'll let you know when it's ready for review

File: main/station-config.js (hunk 1 above)
Comment (jopedroliveira) [reply main/station-config.js:128]: It's a draft. I'll let you know when it's ready for review

**Comment by jopedroliveira** [comment]
> Also this PR is still a draft, shall we mark it ready for review?

It's a draft. I'll let you know when it's ready for review

File: renderer/src/components/WalletTransactionsHistory.tsx (hunk 3 above)
Comment (juliangruber) [reply renderer/src/components/WalletTransactionsHistory.tsx:13]: ok, I just reviewed since you requested a review 👋

**Comment by juliangruber** [comment]
Can you please remove loading from this PR, as we will most likely not need it any more? 🙏

**Review by juliangruber — APPROVED** [review]
nailed it

**Pull request merged**